.github/workflows/test.sh

```diff
@@ -2,8 +2,8 @@
 # manually exiting from script, because after-build needs to run always
 set +e
 
-if [ -z $CC_TEST_REPORT_ID ]; then
-  echo "1. CC_TEST_REPORT_ID is unset, skipping"
+if [ -z $CC_TEST_REPORTER_ID ]; then
```

Comment:
Oh :sweat_smile: :facepalm:

Reply:
It makes much more sense now right 😅  I missed this too

Reply:
Idk how I came up with such variable at first place. I guess I didn't just copy paste.

Reply:
I don't know if I suggested it on a comment or something maybe?

Reply:
Actually you gave the right variable name `CC_TEST_REPORTER_ID` . It was my mistake.